deployments/optimism-goerli/usdc/configuration.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x7E07E15D2a87A24492740D16f5bdF58c16db0c4E",
+  "baseTokenPriceFeed": "0x2636B223652d388721A0ED2861792DA9062D8C73",
+  "borrowMin": "100e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0e15",
+    "baseBorrowSpeed": "0e15",
+    "baseMinForRewards": "1000000e6"
+  },
+  "assets": {
+    "OP": {
+      "priceFeed": "0xE882831E58eec48B7f304482771F67e6b846733D",
```

Comment:
We use the `APE / USD` price feed because there is no `OP / USD` price feed on Optimism-Goerli. Shouldn't a big deal.